README.md

```diff
@@ -20,39 +20,31 @@ The tools in this repository can be built in several different ways:
 If you want to build the tools to use a locally built sourcekitd and SwiftLang, use the Swift repository's build-script to build and test the stress tester by passing `--swiftsyntax` and the desired tools' flags as extra options. To build and run tests, for example, you would run:
```

Comment:
The text here needs an update since you don't need to pass `--swiftsyntax` anymore.